config/config-network.yaml

```diff
@@ -134,7 +134,5 @@ data:
     # This can be used for making Knative report all URLs as "HTTPS" for example, if you're
     # fronting Knative with an external loadbalancer that deals with TLS termination and
     # Knative doesn't know about that otherwise.
+    # AutoTLS scheme semantics are left untouched.
```

Comment:
I'm going to be super annoying now, but just to make sure we explicitly think about this before it's hard to change: should this field be something like "DefaultExternalScheme" since it doesn't really override in this case?

(This way the docs wouldn't need to specify that it gets overridden by autotls but could just say it's the default value when autotls is not set)